src/main/java/org/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -468,6 +468,7 @@ public void setFocusToField(Field field) {
         Field actualField = field;
         boolean fieldFound = false;
             for (Tab tab : tabbed.getTabs()) {
+                tabbed.getSelectionModel().select(tab);
```

Comment:
This looks not right without a condition. What happens with multiple open libraries?
Actually there is a condition below. Why is that not triggered?

Reply:
It works well with multiple opened libraries, just tested.
What condition specifically are you talking about , the very first one inside of for-loop?

Reply:
Answering the question, why that is not triggered, for the reason, that `fieldEditorTab` has a `lazy initialization`, so it doesn't render the fields unless tab opened at least once, that's why I added line 471. I updated description, at the top of this PR, there I described all the changes.

Reply:
Thanks!